playbooks/roles/streisand-gateway/vars/main.yml

```diff
@@ -3,6 +3,7 @@ streisand_gateway_url: "https://{{ streisand_ipv4_address }}"
 
 streisand_gateway_htpasswd_file: "/etc/nginx/htpasswd"
 streisand_gateway_password_file: "/etc/nginx/gateway-password.txt"
+streisand_gateway_password_localpath: "../{{ streisand_local_directory }}/gateway-password.txt"
```

Comment:
If the local fetch isn't scoped to just Vagrant, we would want this file to have some kind of server identifier in its name as well (a la the Shadowsocks password localpath comment).